docs/generate_from_k8s_spec.md

```diff
@@ -0,0 +1,31 @@
+# Generate KCL Packages from Kubernetes OpenAPI Specs
+
+This guide shows how the `k8s` packages are generated. Alternatively, you could skip this guide and directly use the pre-generated Kubernetes packages which are exactly the same as the outcome of this guide. Please refer the [kpm quick start guide](https://github.com/kcl-lang/kpm#quick-start) for how to pull and use the package.
+
+If you want to manually generate them, please continue the guide.
+
+## 1. Download the OpenAPI Spec
+
+Download the Kubernetes OpenAPI Spec from Github. In this guide, we'll generate from the [Kubernetes v1.27 spec](https://github.com/kubernetes/kubernetes/blob/release-1.27/api/openapi-spec/swagger.json)
```

Comment:
Here, you can provide the raw address of the github repository file, which is convenient for direct clicking or executing wget download. e.g., https://raw.githubusercontent.com/kubernetes/kubernetes/release-1.27/api/openapi-spec/swagger.json

Reply:
fixed.